contracts/modules/credit/SpigotedLine.sol

```diff
@@ -163,11 +166,12 @@ contract SpigotedLine is ISpigotedLine, LineOfCredit, ReentrancyGuard {
     }
 
     /**
-     * @notice allows tokens in escrow to be sold immediately but used to pay down credit later
+     * [Bob - why is this function 'detailed' twice?]
```

Comment:
idk, doesnt really matter. just helps with IDE stuff so if we are using external or internal function we still see all the docs.